DependencyInjection/Configuration.php

```diff
@@ -154,6 +154,11 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->end()
                     ->defaultValue([])
                 ->end()
+                ->arrayNode('buckets')
+                    ->prototype('float')->end()
+                    ->defaultValue([0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1.0, 2.5, 5.0, 7.5, 10.0])
```

Comment:
Should we use instead https://github.com/PromPHP/prometheus_client_php/blob/a283aea8269287dc35313a0055480d950c59ac1f/src/Prometheus/Histogram.php#L64
?

Reply:
This introduces a blind space in this bundle. Imagine `promPHP/prometheus-client-php` updates the list of buckets - it may break your user's apps by updating the library without even noticing it. Actually this can be done atm, too.

Defining them inside of your bundle renders it reliable - even on third-party updates like `promPHP/prometheus-client-php`. If you still think we should rely on it, sure - I'll happily make that change.

Reply:
Good point. Instead we should then have a single point of truth for the list in the bundle and refert to `promPHP/prometheus-client-php` and explain the reasons why we defined our own list.